cmd/app/command/commands.go

```diff
@@ -68,7 +68,7 @@ func NewNodeDiskManager() (*cobra.Command, error) {
 	return cmd, nil
 }
 
-// Run ndm
+// RunNodeDiskManager starts the node disk manager
```

Comment:
can you change it like `RunNodeDiskManager starts ndm process`

Reply:
done :)